components/icons/Icons.tsx

```diff
@@ -0,0 +1,84 @@
+import { FC } from 'react';
+import { tw } from '../../utils';
+
+interface IconProps {
+  onClick?: () => void;
+  className?: string;
+}
+
+export const ArrowIcon: FC<IconProps> = ({ onClick, className }) => {
+  return (
+    <svg
+      xmlns='http://www.w3.org/2000/svg'
+      fill='none'
+      viewBox='0 0 24 24'
+      stroke-width='1.5'
+      stroke='currentColor'
+      className={tw('w-6 h-6 cursor-pointer', className)}
+      onClick={onClick}
+    >
+      <path
+        stroke-linecap='round'
+        stroke-linejoin='round'
+        d='M6.75 15.75L3 12m0 0l3.75-3.75M3 12h18'
+      />
+    </svg>
+  );
+};
+
+export const HomeIcon: FC<IconProps> = () => {
+  return (
+    <svg
+      xmlns='http://www.w3.org/2000/svg'
+      fill='none'
+      viewBox='0 0 24 24'
+      stroke-width='1.5'
+      stroke='currentColor'
+      className='w-5 h-5'
```

Comment:
you should be passing the className from props just to keep consistency with ArrowIcon:

```
className={tw('w-6 h-6', className)}
```

also, it seems weird that this icon in particular is of width and height 5 but others are 6, any reasons for that discrepancy?

Reply:
Yes, the smaller ones are used in the sidebar, as they seem smaller to me in the figma.